integration/nwo/fabric/network/network.go

```diff
@@ -236,60 +233,7 @@ func (n *Network) PostRun(load bool) {
 }
 
 func (n *Network) Cleanup() {
-	if n.DockerClient == nil {
-		return
-	}
-
-	nw, err := n.DockerClient.NetworkInfo(n.NetworkID)
-	if _, ok := err.(*docker.NoSuchNetwork); err != nil && ok {
-		return
-	}
-	Expect(err).NotTo(HaveOccurred())
-
-	containers, err := n.DockerClient.ListContainers(docker.ListContainersOptions{All: true})
-	Expect(err).NotTo(HaveOccurred())
-	for _, c := range containers {
-		for _, name := range c.Names {
-			if strings.HasPrefix(name, "/"+n.NetworkID) {
-				logger.Infof("cleanup container [%s]", name)
-				err := n.DockerClient.RemoveContainer(docker.RemoveContainerOptions{ID: c.ID, Force: true})
-				Expect(err).NotTo(HaveOccurred())
-				break
-			} else {
-				logger.Infof("cleanup container [%s], skipped", name)
-			}
-		}
-	}
-
-	volumes, err := n.DockerClient.ListVolumes(docker.ListVolumesOptions{})
-	Expect(err).NotTo(HaveOccurred())
-	for _, i := range volumes {
-		if strings.HasPrefix(i.Name, n.NetworkID) {
-			logger.Infof("cleanup volume [%s]", i.Name)
-			err := n.DockerClient.RemoveVolumeWithOptions(docker.RemoveVolumeOptions{
-				Name:  i.Name,
-				Force: false,
-			})
-			Expect(err).NotTo(HaveOccurred())
-			break
-		}
-	}
-
-	images, err := n.DockerClient.ListImages(docker.ListImagesOptions{All: true})
-	Expect(err).NotTo(HaveOccurred())
-	for _, i := range images {
-		for _, tag := range i.RepoTags {
-			if strings.HasPrefix(tag, n.NetworkID) {
-				logger.Infof("cleanup image [%s]", tag)
-				err := n.DockerClient.RemoveImage(i.ID)
-				Expect(err).NotTo(HaveOccurred())
-				break
-			}
-		}
-	}
-
-	err = n.DockerClient.RemoveNetwork(nw.ID)
-	Expect(err).NotTo(HaveOccurred())
+	// DO nothing
```

Comment:
it looks like that n.DockerClient is still defined but not used anymore, shall we remove?